__tests__/__fixtures__/changelogs/new-docs/new-doc.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Want to rename these directories and files to replace `doc` with `category?` `new-changelog`, `failure-changelogs`, etc

Reply:
Honestly I kinda wanna keep them the same for no other reason than being able to just reuse the subdirectory paths so I can easily make changes to this line when writing tests for custom pages lol
https://github.com/readmeio/rdme/blob/0c644d2d850fdc6e93fba82320c2ca4b427b1bd1/__tests__/cmds/changelogs.test.js#L17